sandbox.go

```diff
@@ -84,6 +84,7 @@ type sandbox struct {
 	ingress            bool
 	ndotsSet           bool
 	oslTypes           []osl.SandboxType // slice of properties of this sandbox
+	loadBalancerNID    string            // NID that this SB is a load balancer for
```

Comment:
isn't this actually the network id?

Reply:
Yes, this is the network ID of the network that this sandbox load balances for.  Of course, this is only set if the sandbox is specifically a load balancer sandbox.  Otherwise a given sandbox is most likely a container and can belong to multiple networks.